src/Dependencies/Roslyn/project.json

```diff
@@ -1,7 +1,8 @@
 {
   "supports": {},
   "dependencies": {
-    "Microsoft.VisualStudio.LanguageServices": "2.0.0-beta5-60909-09"
+    "Microsoft.VisualStudio.LanguageServices": "2.0.0-beta5-60909-09",
+    "System.ValueTuple": "4.0.1-beta-24425-02"
```

Comment:
+@tmat who may know if this is the right version and if this is a safe dependency to add. We need to be sure that it is deployed correctly and that we have a plan in place to use a stable version available to use by RTM.

Consider using another language feature to prove this out that doesn't involve a new dependency.